.vscode/extensions.json

```diff
@@ -1,3 +1,7 @@
 {
-  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode"]
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
+    "esbenp.prettier-vscode",
+    "charliermarsh.ruff"
```

Comment:
Do you use VSCode for Python?

Reply:
Ah yes, maybe I should add that? I kind of thought it was installed by default.

Reply:
Ah just I know people tend to use other IDEs for Python so wanted to make sure you knew this would only work for VSCode :).

Reply:
Wait add what?

Reply:
Ah I thought you meant the default Python VScode extension, I think it has a name like "Python for VScode", or similar.

Anyway, `ruff` is the linter/formatter we use.